### ISSUE TITLE
XYZ axis are not correct

### ISSUE THREAD

**Issue by Kohlbush** [stockto2/desktophero#113]
When rotating or moving bone groups the XYZ axis's don't line up with the displayed blue , green and red lines. Moving seems to be at a diagonal and rotating is just weird. Right now making custom poses is difficult.

**Comment by stockto2** [comment]
I agree - there's a bug here, but it's only with certain certain bone
groups. Arms seem to work correctly from my experience, hands don't.

**Comment by Kohlbush** [comment]
Thanks for responding so quick. I think i figured it out, hands and arms rotate inline with the displayed XYZ. weapons seem to be based off the orientation of the hand, Z is thumb to pinky, X is palm to back hand, and Y is finger to wrist.
